src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newState = Object.assign({}, state);
+  const arrReturn = [];
+
+  for (const obj of actions) {
+    if (obj.type === 'clear') {
```

Comment:
try use switch case when you want to check a value for multiple possible results